src/mailing/sender.js

```diff
@@ -76,11 +76,18 @@ async function sendWelcomeEmail(recipient, user) {
 
 async function sendForgotPasswordEmail(toEmail, resetToken) {
   try {
+    const frontendBaseUrl =
+      process.env.NODE_ENV === 'production'
+        ? 'https://beatbazaar.onrender.com'
+        : 'http://localhost:3000';
```

Comment:
It would be best to put this into an environment variable, so that if we have future cases where we need this value we don't need to rewrite this ternary in other places.

Reply:
thank you , done , commit 54da004c79142c7939b127249727ec46cb324794